packages/template-retail-react-app/app/components/toggle-card/index.jsx

```diff
@@ -34,6 +34,15 @@ export const ToggleCard = ({
     children,
     ...props
 }) => {
+    let editButtonAriaLabel = 'Edit'
+    if (typeof editLabel === 'string') {
+        editButtonAriaLabel = editLabel
+    } else if (
+        typeof editLabel === 'object' &&
+        editLabel?.type?.displayName === 'MemoizedFormattedMessage'
+    ) {
+        editButtonAriaLabel = editLabel?.props.defaultMessage[0].value
+    }
```

Comment:
It is not necessary to have this if logic here, any reason we can't consume editLabel prop directly into the component as 
```
// ToggleCart.jsx
...
  <Button
       variant="link"
       size="sm"
        onClick={onEdit}
       aria-label={editLabel}
/>
...


// Payment Component
<ToggleCart  editLabel={intl.formattedMessage({id: 'Payment Edit', defaultMessage: "Payment Edit"})
  
```

Reply:
Replaced logic passing formatMessage() to editLabel